data/fields/artwork_type.json

```diff
@@ -12,6 +12,10 @@
             "bust": "Bust",
             "painting": "Painting",
             "mosaic": "Mosaic",
+            "tilework": "Tile work",
+            "stone": "Stone",
+            "dwarf": "Dwarf",
```

Comment:
this is in use but highly local, orthogonal to other values and confusing

I highly oppose this confusing inclusion

Reply:
For example: statue of dwarf, mosaic depicting dwarf etc.

Suddenly tagging all statues in one way in this key - with exception of dwarf statues - is deeply silly and confusing.

It is also missing from https://wiki.openstreetmap.org/wiki/Key:artwork_type

See also https://github.com/osmlab/name-suggestion-index/issues/10158

Reply:
I have no hard feelings there, I just took some of the top-used values. Removed `dwarf` from the list.